lib/requests.js

```diff
@@ -149,6 +158,12 @@ QueryRequest.prototype.write = function (encoder) {
   //v3: <query>
   //      <consistency><flags>[<n>[name_1]<value_1>...[name_n]<value_n>][<result_page_size>][<paging_state>][<serial_consistency>][<timestamp>]
   var frameWriter = new FrameWriter(types.opcodes.query);
+  var headerFlags = this.options.traceQuery ? types.frameFlags.tracing : 0;
```

Comment:
I was wondering the same the thing when I implemented it. There is no way to deal with prepared statement data in the Node.js driver, it is handled by the driver and abstracted away.

I think it would make sense in case we add a method to `prepareAll()` or something like that, that will enable users to prepare statements before the first execution, manually (most likely on app start), but not for now.